packages/components/src/theme-provider/index.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import { ThemeProvider } from 'emotion-theming';
+
+export { useTheme as useTheme } from 'emotion-theming';
+
+const getTheme = ( theme ) => ( {
+	fontSizes: {
+		small: 11,
+		default: 13,
+		medium: 18,
+		big: 22,
+	},
+	space: {
+		small: 4,
+		medium: 8,
+		large: 16,
+		xlarge: 24,
+		iconButtonSize: 36,
+		iconButtonSizeSmall: 24,
+		borderWidth: 1,
+	},
+	zIndices: {
+		'block-library-gallery-item__inline-menu': 20,
+	},
+	colors: themes[ theme || '' ] || themes.default,
+	helpers: {
+		shade: ( inputColor, amount ) => {
+			return '#' + inputColor.replace( /^#/, '' ).replace( /../g, ( color ) => ( '0' + Math.min( 255, Math.max( 0, parseInt( color, 16 ) + amount ) ).toString( 16 ) ).substr( -2 ) );
+		},
+	},
+} );
+
+const themes = {
+	default: {
+		primary: '#0085ba',
+		secondary: '#11a0d2',
+		toggle: '#11a0d2',
+		button: '#0085ba',
+		outlines: '#007cba',
+		'dark-gray-300': '#6c7781',
+		'dark-gray-500': '#555d66',
+		'dark-gray-900': '#191e23',
+		'light-gray-500': '#e2e4e7',
+		white: '#ffffff',
+		alertRed: '#d94f4f',
+	},
+	'admin-color-light': {
```

Comment:
Neat! Adding values to the base theme. I noticed that we already have the values from `@wordpress/base`:

https://github.com/WordPress/gutenberg/blob/master/packages/base-styles/index.js

What are your thoughts on importing those values and extending this theme object? That way, we won't have to repeat the values :)

Reply:
I'm totally fine with importing the `base-styles` here. I created this `theme-provider` during the `experimental x-platform Box` a few months ago and haven't changed it since then.